### ISSUE TITLE
[Voice access - Adaptive Cards]: User is not able to select the 'Time hours and Minutes' under Expended "Timing" combo box using VA Commands.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7930]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:

OS Build: 25201.1000
Edge dev: Version 107.0.1379.1 (official build) dev (64-bit)

Application URL: [Schema Explorer | Adaptive Cards](https://adaptivecards.io/explorer/)

Accessibility Tool: Voice Access

Pre-requisite:
OS Settings->Accessibility->Voice Access (Turn ON)

Repro Steps:
Launch the above-mentioned URL on edge Dev with Valid credentials.
Command: "Voice access wake up"
Command: "Click Time".
Command: "Click Time".
Observe 2no's are given for the time components select the Appropriate no from those 2no's as 2.
Now try to change the timing from the "What time do you want to meet" combo box using VA commands.
Observe the issue.
Actual Result:
User is unable to change the timing from the "What time do you want to meet" combo box using VA commands.
Expended "Timing" combo box 'time hours and minutes' control is not accessible using VA click Commands.

Note:
Similar issue present at "input.date" link.
PFA: Date issue.mp4

Expected Result:
User Should be able to edit the timing from the "What time do you want to meet" combo box using VA commands.

User Experience:
Voice access users will get impact and will get disoriented if this behavior persists.

MAS Reference:
[No Disruption of Accessibility Features - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.03.01)

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://user-images.githubusercontent.com/104482612/192790904-bb325000-d4ab-442f-af3c-3f5fa449a3f3.mp4

https://user-images.githubusercontent.com/104482612/192790937-596b885f-1609-46e1-a244-c21e58a6240b.mp4

![code snippet of date](https://user-images.githubusercontent.com/104482612/198583155-ecce2a34-2bbe-47b0-8d65-6ce7a7768cf6.png)
<img width="960" alt="code snippet of time" src="https://user-images.githubusercontent.com/104482612/198583177-490e99b8-368e-4ec1-b67e-80a614bda112.png">

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @Susmitha474,

Our team uses HTML's `<input type="date"/>` and `<input type="time"/>` for these two controls. Because of this, we cannot make any modifications to the contents of the Calendar picker or Time picker.

The command "Show numbers" appears to be an issue within Edge, and does not repro on other browsers.

**Test HTML**
```HTML
<div class="ac-container ac-adaptiveCard" style="display: flex; flex-direction: column; justify-content: flex-start; box-sizing: border-box; flex: 0 0 auto; padding: 10px; margin: 0px; background-color: rgb(255, 255, 255);"><div style="display: flex; flex-direction: column; box-sizing: border-box; flex: 0 0 auto;"><div class="ac-input-container" style="display: flex;"><input type="date" tabindex="0" class="ac-input ac-dateInput" id="__ac-2a000960-bf60-44e5-a608-2dd68ad955f6" style="width: 100%; min-width: 0px;"></div></div><div class="ac-horizontal-separator" aria-hidden="true" style="height: 8px; overflow: hidden; flex: 0 0 auto; display: flex; margin-right: 0px; margin-left: 0px;"></div><div style="display: flex; flex-direction: column; box-sizing: border-box; flex: 0 0 auto;"><div class="ac-input-container" style="display: flex;"><input type="time" class="ac-input ac-timeInput" tabindex="0" id="__ac-f07307d8-a390-417f-bbba-789ae3ac9e97" style="width: 100%; min-width: 0px;"></div></div></div>
```

**Edge Voice Access**
![edge](https://user-images.githubusercontent.com/98650930/201181467-b50a78b5-c957-4dc2-94ad-79f6e41b5efa.gif)

**Chrome Voice Access**
![chrome](https://user-images.githubusercontent.com/98650930/201181488-01102bfe-7f8d-401d-a4eb-ecf437c6b7b2.gif)

Please let me know if you have any questions or if there is anything else we can do to help resolve this issue. Thanks!

**Comment by vagpt** [comment]
Reactivating this issue as per the comment below from the product team.

This is mainly UIA issue

**Snippet:**
![image](https://user-images.githubusercontent.com/95913748/213976620-c3799ed5-14ed-4dbc-bfb0-274801d5939d.png)

There are multiple reasons why the control will not be clicked or focused or why even show numbers also won't work.

1. Name is an empty string ("") & even Value.Value property doesn't exist.
2. Control type is group control, which we have not included in interactable control types list.
3. It's not keyboard focusable.

**Comment by anna-dingler** [comment]
After syncing with the accessibility team, I am resolving this issue because it is being transferred to the Edge team.
